src/libs/SidebarUtils.ts

```diff
@@ -170,7 +170,6 @@ function getOrderedReportIDs(
 
     // The LHN is split into five distinct groups, and each group is sorted a little differently. The groups will ALWAYS be in this order:
     // 1. Pinned - Always sorted by reportDisplayName
-    // 2. Outstanding IOUs - Always sorted by iouReportAmount with the largest amounts at the top of the group
     // 3. Drafts - Always sorted by reportDisplayName
     // 4. Non-archived reports and settled IOUs
```

Comment:
Nitpick but can you update the numbering and the description for 1 (or add in a modified `2` based on my other question)

Reply:
Yep.